week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -1,11 +1,18 @@
 /*
 Write a function that:
 - takes an array of strings as input
-- removes any spaces in the beginning or end of the strings
-- removes any forward slashes (/) in the strings
+- removes any spaces in the beginning or end of the strings trim
+- removes any forward slashes (/) in the strings replace
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString(strArr) {
+  for (let i = 0; i < strArr.length; i++) {
+    strArr[i] = strArr[i].trim();
+    strArr[i] = strArr[i].replace(/\//g, "");
+    strArr[i] = strArr[i].toLowerCase();
+  }
+  return strArr;
+}
 
```

Comment:
Another way to write this

Is to change Line 10) - 12)
```js
strArr[i] = strArr[i].trim();
strArr[i] = strArr[i].replace(/\//g, "");
strArr[i] = strArr[i].toLowerCase();
```
To just one line. This is the POWER of function chaining - and functional programming
```js
strArr[i] = strArr[i].trim().replace(/\//g, "").toLowerCase();
```

To using Function chaining